episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+continue using dark mode if you prefer.
+
+This histogram summarises the pixel values of the entire image. On the x axis 
+is the pixel value which run from 0-255 for this 8-bit image. This is split 
+into a number of 'bins' of a certain width (for example, it could be 0-10, 11-20 
+and so on...). Each bin has a blue bar whose height is proprotional to the 
+number of pixels with values in that bin. So, for example, for our mitosis image 
+we see the highest bars to the left, with shorter bars to the right. This means 
+this image has a lot of very dark (low intensity) pixels and fewer bright 
+(high intensity) pixels.
+
+Let's quickly compare to another image. Open the 'coins' image with:  
+`File > Open Sample > napari builtins > Coins`
+
+![](fig/coins-histogram.png
+){alt="Screenshot of image histogram for coins image" width='68%'}
+
+From the histogram, we can see that this image has a wider spread of pixel 
+values. There are bars of similar height across many different values (rather 
+than just one big peak at the left hand side).
+
+Image histograms are a great way to quickly summarise and compare pixel values 
+of different images.
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Histograms
+
+Match each of these small test images to their corresponding histogram. You can 
+assume that all images are displayed with a standard gray colormap and default 
+contrast limits of the min/max possible pixel values:
+
+![](fig/exercise-hist-images.png
+){alt="Screenshot of 4 small grayscale test images" width='68%'}
+
+![](fig/exercise-histograms.png
+){alt="Screenshot of 4 histograms, corresponding to the test images"}
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+- a - 3
+- b - 4
+- c - 2
+- d - 1
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Changing display settings
+
+What happens to pixel values when we change the display settings? Try changing 
+the contrast limits or colormap in the layer controls. You should see that the 
+histogram stays the same, no matter what settings you change i.e. the display 
+settings don't affect the underlying pixel values.
+
+This is one of the reasons it's important to use software designed for 
+scientific analysis to work with your light microscopy images. Software like 
+Napari and ImageJ will try to ensure that your pixel values remain unchanged, 
+while other image editing software (designed for working with photographs) may 
+change the pixel values in unexpected ways. Also, even with scientific software, 
+some image processing steps (that we'll see in later episodes) will change the 
+pixel values. 
+
+Keep this in mind and make sure you always retain a copy of your original data, 
+in its original file format! We'll see in episode X that original image files 
+contain important metadata that should be kept for future reference.
+
+## Colormaps / LUTs
+
+Let's dig deeper into Napari's colormaps. We've already seen in episode X that 
```

Comment:
I don't think we have really explained what a colormap is yet, so I think it might be worth looping back to the start of the image, where the same voxels are displayed with different colors. I think it might be worth a reminder here to start here that pixels just hold numbers and we have to find a way that best displays these numbers as pixel intensities and colours. Enter the colormap. And then use some of the text below.